src/components/textarea/__tests__/textarea.test.tsx

```diff
@@ -1,19 +1,40 @@
+import isCSSFieldSizingSupported from '../isCSSFieldSizingSupported'
 import { render, screen } from '@testing-library/react'
 import { TextArea } from '..'
 
+jest.mock('../isCSSFieldSizingSupported')
+
 test('default min/max rows are 2 -> Infinity', () => {
+  jest.mocked(isCSSFieldSizingSupported).mockReturnValue(true)
+
   const { asFragment } = render(<TextArea />)
   expect(asFragment()).toMatchSnapshot()
 })
 
 test('can set custom min/max rows', () => {
+  jest.mocked(isCSSFieldSizingSupported).mockReturnValue(true)
+
   const { asFragment } = render(<TextArea maxRows={10} minRows={3} />)
   expect(asFragment()).toMatchSnapshot()
 })
 
 // TODO: This test is currently skipped because our Linaria styled global mock
 // is changing the tag name.
-test.skip('is accessible via the textbox role', () => {
-  render(<TextArea maxRows={10} minRows={3} />)
+test.skip('is always accessible via the textbox role', () => {
```

Comment:
Yeah, if I get some time, will work out if that mock is still required. _I think_ not but will validate.

Reply:
I've carved out time this sprint to take a look at this too. A quick check suggests they're no longer required, but I need to take a closer look as my gut says that's only part of the problem.